hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/PauseUnpauseTokenAccountPrecompileSuite.java

```diff
@@ -88,7 +113,263 @@ protected Logger getResultsLogger() {
 
     @Override
     public List<HapiSpec> getSpecsInSuite() {
-        return List.of(noTokenIdReverts(), noAccountKeyReverts());
+        return List.of(
+                noTokenIdReverts(),
+                noAccountKeyReverts(),
+                pauseFungibleTokenHappyPath(),
+                unpauseFungibleTokenHappyPath(),
+                pauseNonFungibleTokenHappyPath(),
+                unpauseNonFungibleTokenHappyPath());
```

Comment:
The test names ending in "HappyPath" may suggest that they cover only positive scenarios.
For clarity and better understanding, maybe we could consider more descriptive names that highlight the scenarios being tested.